### PR TITLE
Fixes https://github.com/square/keywhiz-fs/issues/15

### PR DESCRIPTION
**Pull request by alokmenghrajani**
- [x] @csstaub 
- [ ] @mcpherrinm

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -58,7 +59,7 @@ type httpClientParams struct {
 
 // NewClient produces a read-to-use client struct given PEM-encoded certificate file, key file, and
 // ca file with the list of trusted certificate authorities.
-func NewClient(certFile, keyFile, caFile, serverURL string, timeout time.Duration, logConfig klog.Config, ping bool) (client Client) {
+func NewClient(certFile string, keyFile string, caFile string, serverURL *url.URL, timeout time.Duration, logConfig klog.Config, ping bool) (client Client) {
```
Comment (csstaub) [inline client.go]: Use `certFile, keyFile, caFile string`

**Comment by csstaub** [comment]
:+1:

File: client.go (hunk 1 above)
Comment (alokmenghrajani) [inline client.go]: what's the reasoning? having each type seems more readable.

**Comment by mcpherrinm** [comment]
:+1:

**Pull request merged**